test/task/SetLocalDescriptionTask.test.ts

```diff
@@ -59,21 +60,21 @@ describe('SetLocalDescriptionTask', () => {
     it('can be run and succeed', async () => {
       await task.run();
       const peerLocalSDP = context.peer.localDescription.sdp;
-      expect(peerLocalSDP).to.be.equal(sdpOffer.sdp);
+      expect(peerLocalSDP).to.be.equal(sdpOffer.sdp + CRLF);
```

Comment:
previously the SDP was not recreated, and the original was missing this line ending.